### DISCUSSION TITLE
Export Control Classification Number (ECCN) for RabbitMQ Server?

### DISCUSSION THREAD

**Discussion by f-camera** [rabbitmq/rabbitmq-server#5411] [Other]
Hi All,
I'm in search for ECCN code for RabbitMQ Server. Currently I'm packaging Erlang/OTP and RabbitMQ Server together with my commercial product and delivering them to customers. I deliver both pre-built binaries and source code downloaded from (e.g.) [https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.10.6](https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.10.6) .
Could you please help understanding how RabbitMQ Server can be classified about ECC?

Many thanks.

**Comment by michaelklishin** [comment]
I don't see a reason why RabbitMQ — a piece of open source software that does not itself include an implementation of cryptographic functions or any other "sensitive" bits — would require a license to be exported.

ECCN for products exported without a license [is documented to be EAR99](https://www.bis.doc.gov/index.php/licensing/commerce-control-list-classification/export-control-classification-number-eccn).
